src/Uno.UI.Tests/BinderTests/Given_Binder.inpc.cs

```diff
@@ -0,0 +1,225 @@
+				new Binding
+				{
+					Path = new PropertyPath("Class1.Value")
+				});
+
+			SUT.SetBinding(
+				Binder_INPC_Data.MyValue2Property,
+				new Binding
+				{
+					Path = new PropertyPath("Value"),
+					Converter = new Binder_INPC_DummyConverter()
+				});
+
+			var master = new Binder_INPC_Base_Class();
+			SUT.DataContext = master;
+
+			master.RaiseAllUpdated();
+
+			Assert.AreEqual(SUT.MyValue, master.Class1.Value);
+			Assert.AreEqual(SUT.MyValue2, master.Value);
+		}
+	}
+
+	public partial class Binder_INPC_Data : DependencyObject
+	{
+		public string MyValuePropertyValueDuringChange { get; private set; }
+
+		public string MyValue
+		{
+			get => GetMyValueValue();
+			set => SetMyValueValue(value);
+		}
+
+		[GeneratedDependencyProperty(DefaultValue = "")]
+		public static DependencyProperty MyValueProperty { get; } = CreateMyValueProperty();
+
+		private void OnMyValueChanged(string oldValue, string newValue)
+		{
+			MyValuePropertyValueDuringChange = MyValue;
+		}
+
+		public string MyValue2
+		{
+			get => GetMyValue2Value();
+			set => SetMyValue2Value(value);
+		}
+
+		[GeneratedDependencyProperty(DefaultValue = "")]
+		public static DependencyProperty MyValue2Property { get; } = CreateMyValue2Property();
+	}
+
+	public class Binder_INPC_Base_Class : Binder_INPC_BaseViewModel
+	{
+		private string _value = Guid.NewGuid().ToString();
+		public int ValueGetCount { get; private set; }
+		public int ValueSetCount { get; private set; }
+
+		public Binder_INPC_Class1 Class1 { get; set; } = new Binder_INPC_Class1();
+		public string Value
+		{
+			get
+			{
+				ValueGetCount++;
+				return _value;
+			}
+			set
+			{
+				ValueSetCount++;
+				_value = value;
+			}
+		}
+
+		public void Update()
+		{
+			Class1.Value = (Binder_INPC_Class1.IndexCounter++).ToString();
+			Value = (Binder_INPC_Class1.IndexCounter++).ToString();
+			RaiseUpdated();
+		}
+
+		public void RaiseUpdated()
+		{
+			OnPropertyChanged(nameof(Class1));
+			OnPropertyChanged(nameof(Value));
+		}
+
+		public void RaiseAllUpdated()
+		{
+			Class1.Value = (Binder_INPC_Class1.IndexCounter++).ToString();
+			Value = (Binder_INPC_Class1.IndexCounter++).ToString();
+			OnPropertyChanged(string.Empty);
+		}
+	}
+
+	public class Binder_INPC_Class1
+	{
+		public static int IndexCounter = 0;
+		private string _value = (IndexCounter++).ToString();
+		public int ValueGetCount;
+		public int ValueSetCount;
+
+		public string Value
+		{
+			get
+			{
+				ValueGetCount++;
+				return _value;
+			}
+			set
+			{
+				ValueSetCount++;
+				_value = value;
+			}
+		}
+	}
+
+	public class Binder_INPC_BaseViewModel : INotifyPropertyChanged
+	{
+		public event PropertyChangedEventHandler PropertyChanged;
+
+		protected void OnPropertyChanged([CallerMemberName] string propertyName = "")
+			=> PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+	}
+
+	public class Binder_INPC_DummyConverter : IValueConverter
+	{
+		public static int ConvertCount;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5775486)